docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -1,61 +1,35 @@
 .. _network-best-practices:
 
-**************************************
-Network Best Practices for Ansible 2.5
-**************************************
+************************
+Ansible Network Examples
+************************
 
+This document describes some examples of using Ansible to manage your network infrastructure.
 
-Overview
-========
-
-This document explains the best practices for using Ansible 2.5 to manage your network infrastructure.
-
-
-Audience
---------
-
-* This example is intended for network or system administrators who want to understand how to use Ansible to manage network devices.
-
+.. contents::
+   :local:
 
 Prerequisites
--------------
+=============
 
 This example requires the following:
 
-* **Ansible 2.5** (or higher) installed. See :doc:`../../installation_guide/intro_installation` for more information.
+* **Ansible 2.5** (or higher) installed. See :ref:`intro_installation_guide` for more information.
 * One or more network devices that are compatible with Ansible.
-* Basic understanding of YAML :doc:`../../reference_appendices/YAMLSyntax`.
-* Basic understanding of Jinja2 Templates. See :doc:`../../user_guide/playbooks_templating` for more information.
+* Basic understanding of YAML :ref:`yaml_syntax`.
```

Comment:
I actually like the links to the other pages in case they aren't familiar with YAML.

Reply:
The link still works - this is just the preferred way to crosslink within the same docsite for ansible docs.

Reply:
thanks for the catch! didn't noticed I'd deleted a link

Reply:
ooph.. the link is there in the doc snippet below this.